File: klijent/Mec.js
export class Mec {

    constructor(domacin,gost,poenidomacin,poenigost,kolo,sudija,sezona) {
        this.domacin = domacin;
        this.gost = gost;
        this.poenidomacin = poenidomacin;
        this.kolo = kolo;
        this.poenigost = poenigost;
        this.sudija=sudija;
        this.sezona=sezona;
    }

    crtaj(host) {

        var tr = document.createElement("tr");
        host.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML = this.domacin.ime;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.poenidomacin;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.poenigost;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.gost.ime;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.kolo;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.sudija.ime + " " + this.sudija.prezime;
        tr.appendChild(el);
    }
}

File: server/models/Mec.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Collections.Generic;
using System.Text.Json.Serialization;
using System;

namespace Models
{

    [Table("Mec")]
    public class Mec
    {
        // Ovde pisemo propertije
        [Key]
        public int MecID { get; set; }

        [Required]
        public Sezona Sezona { get; set; }

        [Required]
        public int Kolo { get; set; }
        
        [Required]
        public Klub Domacin { get; set; }
        
        [Required]
        public Klub Gost { get; set; }

        [Required]
        public int Poenidomacin { get; set; }

        [Required]
        public int Poenigost { get; set; }

        [Required]
        public Sudija Sudija { get; set; }
    }
}